configs/cornernet/cornernet_hourglass104_aug_test_batch3.py

```diff
@@ -0,0 +1,117 @@
+_base_ = ['../_base_/default_runtime.py']
```

Comment:
The config filenames should follow the [naming convention](https://mmdetection.readthedocs.io/en/latest/config.html#config-name-style).